tests/functests/instancetype_test.go

```diff
@@ -181,12 +181,12 @@ var _ = Describe("Common instance types func tests", func() {
 			Expect(err).ToNot(HaveOccurred())
 		})
 
-		DescribeTable("a Linux guest with", func(containerDisk string, preferences map[string]string, testFns []testFn) {
+		DescribeTable("a Linux guest with", func(containerDisk, instanceType string, preferences map[string]string, testFns []testFn) {
```

Comment:
Wont the tests in L146 fail without a larger instancetype?

Reply:
Yes, fixed.